addons/web_editor/static/lib/odoo-editor/test/spec/align.test.js

```diff
@@ -0,0 +1,316 @@
+import { BasicEditor, testEditor } from '../utils.js';
```

Comment:
What is this "spec" folder? Why "spec"?

Reply:
It's for separating the testing files from the code that launch the tests and the utility file.

`spec` mean `specification`. This term is also broadly used to reference a testing file.
I've used it to avoid `test/test`.